src/main/python/cfn_sphere/stack_configuration/__init__.py

```diff
@@ -9,22 +9,20 @@
 class Config(object):
     def __init__(self, config_file=None, config_dict=None, cli_params=None):
 
-        if config_dict:
-            self.dict = config_dict
+        if isinstance(config_dict, dict):
```

Comment:
Yes, although I did not change the business logic in this function but rather the way it is implemented. Actually [Lab Manager Light](https://github.com/ImmobilienScout24/lab-manager-light/blob/master/src/lml/lib/LML/Config.pm#L8) also used this trick so it did not surprise me. It helps a lot in writing tests :-)